AdobeStockImageAdminUi/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="system">
+            <group id="adobe_stock_integration" translate="label" type="text" sortOrder="1100" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
I think these settings make sense only on a global level, as they are used to configure admin panel functionality, so `showInWebsite` and `showInStore` should be `0` for all of them

Reply:
make sense